### ISSUE TITLE
fix color contrast in headers and footers for accessibility

### ISSUE THREAD

**Issue by brainwane** [solutionguidance/psm#514]
From HTML CodeSniffer/`pa11y`, we have a few color contrast issues with headers, footers, etc. Example:

* "Contributions welcome" footer: "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.4:1. Recommendation: change text colour to #004f7d." `WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail`
(`<a href="https://github.com/OpenTechStrategies/psm">Contributions welcome</a>`)

* Logout link in header: "This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.93:1. Recommendation: change background to #033a6e." (`...<a href="javascript:;">Help</a><a href="/cms/j_spring_security_logout">Logout</a> `)

Success Criterion:
[1.4.3: Contrast (Minimum)](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast)
Suggested Techniques:
[G18](http://www.w3.org/TR/WCAG20-TECHS/G18)

**Comment by PaulMorris** [comment]
This was fixed with commit 799d05b808cec13f61604999d9dad2e02015c98e (which was accidentally pushed directly to master).  The changes are subtle but enough to fix the accessibility problem.  Here are some screenshots:

Before:
![screenshot-2017-11-28-dashboard-before](https://user-images.githubusercontent.com/1091693/33353976-8e595856-d47e-11e7-9e9d-3660bc297cf8.png)
After:
![screenshot-2017-11-28-dashboard-after](https://user-images.githubusercontent.com/1091693/33353986-9a7fabbc-d47e-11e7-8079-66505560c6bb.png)
